### PR TITLE
EDU-2436: Update security coverage of PrivateLink for accuracy in wording

### PR DESCRIPTION
**Pull request by fairlydurable**
Update security coverage of PrivateLink for accuracy in wording. 

Source of truth: Raphaël Beamonte

### CODE REVIEW

**Review by flossypurse — APPROVED** [review]
Once you get sign off from SME, lgtm.

**Review by XaF — COMMENTED** [review]
Thanks a lot!

**Pull request merged**
